optic.md

```diff
@@ -42,17 +42,27 @@ with Optic.
 "optic:capture": "dotenv -e .env optic capture openapi.yml --server-override $baseUrl --update interactive"
 ```
 
-Initialize the capture configuration. This creates an `optic.yml` file
+Initialize the capture configuration. This creates an `optic.yml` file. You need to install @useoptic/optic globally for this, and it is a one time operation.
 
 ```bash
 optic capture init openapi.yml
 ```
```

Comment:
Since this is already copied into the repo, you might not need to reinitialize the capture init

Reply:
I will specify doing this once, and configuring it per your suggestions.